bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "legislative-simulation",
+  "version": "0.0.0",
+  "homepage": "https://github.com/jtoney1/legislative-simulation",
+  "authors": [
+    "james <jatoney@hotmail.com>"
+  ],
+  "license": "MIT",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
```

Comment:
Ah, I see that now. We might want to look into bundling it with our app in order to reduce the number of requests, but I think for now it should be fine. It's something that can be added later pretty easily if we decide to go that route.